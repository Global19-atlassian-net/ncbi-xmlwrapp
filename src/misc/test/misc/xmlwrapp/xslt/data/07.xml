<build_outputs>
  <build_output>
    <build_run>52757</build_run>
    <task_type>genome_align_alt</task_type>
    <plane>asm2asm</plane>
    <task_name>Align alternate loci</task_name>
    <role>alt_to_ref_align</role>
    <batch_id>757491</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/52757/genome_align_alt.557472/out/alt_to_ref_align.seqalign.asnb</path>
      <name>alt_to_ref_align.seqalign.asnb</name>
    </out_file>
  </build_output>
  <build_output>
    <build_run>52757</build_run>
    <task_type>genome_align</task_type>
    <plane>asm2asm</plane>
    <task_name>Align assemblies</task_name>
    <role>review</role>
    <batch_id>764301</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/52757/genome_align.557462/out/reports/</path>
      <name>4 Files</name>
    </out_file>
  </build_output>
  <build_output>
    <build_run>53467</build_run>
    <task_type>gnomon_evidence_summary</task_type>
    <plane>gnomon-full</plane>
    <task_name>Summarize Gnomon Evidence</task_name>
    <role>full_report</role>
    <batch_id>772691</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_report.txt</path>
      <name>gnomon_report.txt</name>
    </out_file>
  </build_output>
  <build_output>
    <build_run>53467</build_run>
    <task_type>gnomon_evidence_summary</task_type>
    <plane>gnomon-full</plane>
    <task_name>Summarize Gnomon Evidence</task_name>
    <role>summary_bags</role>
    <batch_id>772701</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_bag_summary.txt</path>
      <name>gnomon_bag_summary.txt</name>
    </out_file>
  </build_output>
  <build_output>
    <build_run>53467</build_run>
    <task_type>gnomon_evidence_summary</task_type>
    <plane>gnomon-full</plane>
    <task_name>Summarize Gnomon Evidence</task_name>
    <role>summary_pairs</role>
    <batch_id>772711</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_pair_summary.txt</path>
      <name>gnomon_pair_summary.txt</name>
    </out_file>
  </build_output>
  <build_output>
    <build_run>53467</build_run>
    <task_type>gnomon_evidence_summary</task_type>
    <plane>gnomon-full</plane>
    <task_name>Summarize Gnomon Evidence</task_name>
    <role>summary_stats</role>
    <batch_id>772721</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_stats_summary.txt</path>
      <name>gnomon_stats_summary.txt</name>
    </out_file>
  </build_output>
  <build_output>
    <build_run>53467</build_run>
    <task_type>gnomon_evidence_summary</task_type>
    <plane>gnomon-full</plane>
    <task_name>Summarize Gnomon Evidence</task_name>
    <role>summary_taxids</role>
    <batch_id>772731</batch_id>
    <out_file>
      <path>/panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_taxid_summary.txt</path>
      <name>gnomon_taxid_summary.txt</name>
    </out_file>
  </build_output>
</build_outputs>
