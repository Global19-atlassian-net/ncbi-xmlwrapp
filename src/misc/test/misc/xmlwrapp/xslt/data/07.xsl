<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- Edited by XMLSpy® -->
<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">

<xsl:template match="build_outputs">
    <table border="1" width="800" cellspacing="1" cellpadding="1" >
      <tr bgcolor="#66CCFF">
        <th>Build Run</th>
        <th>Task type</th>
        <th>Plane</th>
        <th>Task Name</th>
        <th>Role</th>
        <th>Path</th>
      </tr>
      <xsl:for-each select="build_output">
      <tr>
        <td><xsl:value-of select="build_run"/></td>
        <td><xsl:value-of select="task_type"/></td>
        <td><xsl:value-of select="plane"/></td>
        <td><xsl:value-of select="task_name"/></td>
        <td><xsl:value-of select="role"/></td>
        <td>
          <xsl:choose>
          <xsl:when test="count(out_file)>0">
            <a href="file:/{out_file/path}" target="_blank"> 
              <xsl:value-of select="out_file/name"/>
            </a>
          </xsl:when>
          <xsl:otherwise>No Output.</xsl:otherwise>
          </xsl:choose>
        </td>
      </tr>
      </xsl:for-each>
    </table>
</xsl:template>
</xsl:stylesheet>


