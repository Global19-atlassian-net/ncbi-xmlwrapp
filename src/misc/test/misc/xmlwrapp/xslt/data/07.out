<table border="1" width="800" cellspacing="1" cellpadding="1">
  <tr bgcolor="#66CCFF">
    <th>Build Run</th>
    <th>Task type</th>
    <th>Plane</th>
    <th>Task Name</th>
    <th>Role</th>
    <th>Path</th>
  </tr>
  <tr>
    <td>52757</td>
    <td>genome_align_alt</td>
    <td>asm2asm</td>
    <td>Align alternate loci</td>
    <td>alt_to_ref_align</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/52757/genome_align_alt.557472/out/alt_to_ref_align.seqalign.asnb" target="_blank">alt_to_ref_align.seqalign.asnb</a>
    </td>
  </tr>
  <tr>
    <td>52757</td>
    <td>genome_align</td>
    <td>asm2asm</td>
    <td>Align assemblies</td>
    <td>review</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/52757/genome_align.557462/out/reports/" target="_blank">4 Files</a>
    </td>
  </tr>
  <tr>
    <td>53467</td>
    <td>gnomon_evidence_summary</td>
    <td>gnomon-full</td>
    <td>Summarize Gnomon Evidence</td>
    <td>full_report</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_report.txt" target="_blank">gnomon_report.txt</a>
    </td>
  </tr>
  <tr>
    <td>53467</td>
    <td>gnomon_evidence_summary</td>
    <td>gnomon-full</td>
    <td>Summarize Gnomon Evidence</td>
    <td>summary_bags</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_bag_summary.txt" target="_blank">gnomon_bag_summary.txt</a>
    </td>
  </tr>
  <tr>
    <td>53467</td>
    <td>gnomon_evidence_summary</td>
    <td>gnomon-full</td>
    <td>Summarize Gnomon Evidence</td>
    <td>summary_pairs</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_pair_summary.txt" target="_blank">gnomon_pair_summary.txt</a>
    </td>
  </tr>
  <tr>
    <td>53467</td>
    <td>gnomon_evidence_summary</td>
    <td>gnomon-full</td>
    <td>Summarize Gnomon Evidence</td>
    <td>summary_stats</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_stats_summary.txt" target="_blank">gnomon_stats_summary.txt</a>
    </td>
  </tr>
  <tr>
    <td>53467</td>
    <td>gnomon_evidence_summary</td>
    <td>gnomon-full</td>
    <td>Summarize Gnomon Evidence</td>
    <td>summary_taxids</td>
    <td>
      <a href="file://panfs/pan1/gpipe/prod/data01/Sus_scrofa/3.1.12644/53467/gnomon_evidence_summary.564052/out/gnomon_taxid_summary.txt" target="_blank">gnomon_taxid_summary.txt</a>
    </td>
  </tr>
</table>

